packages/sanes-chrome-extension/src/routes/recovery-phrase/index.dom.spec.ts

```diff
@@ -1,99 +1,61 @@
-      txs: [],
-    };
-    jest.spyOn(chromeInternalMsgs, 'createPersona').mockResolvedValue(createPersonaResponse);
-
-    const getPersonaResponse: GetPersonaResponse = {
-      accounts: [],
-      mnemonic,
-      txs: [],
-    };
-    jest.spyOn(chromeInternalMsgs, 'getPersonaData').mockResolvedValue(getPersonaResponse);
-  });
-
-  it('has a back button that redirects to the previous route when clicked', async (): Promise<void> => {
+  it('has a back button that redirects to the previous route when clicked', async () => {
     await travelToAccount(store);
     await travelToRecoveryPhrase(store);
     expect(backButton.getAttribute('aria-label')).toBe('Go back');
-    TestUtils.act(
-      (): void => {
-        TestUtils.Simulate.click(backButton);
-      },
-    );
+    click(backButton);
     await whenOnNavigatedToRoute(store, ACCOUNT_STATUS_ROUTE);
-  });
+  }, 60000);
 
   it('has an "Export as .PDF" button', () => {
     expect(exportButton.textContent).toBe('Export as .PDF');
-  });
+    //TODO test downloads pdf
+  }, 60000);
 
   it('shows an empty mnemonic if there is no current Persona', () => {
-    const mnemonic = TestUtils.scryRenderedDOMComponentsWithTag(recoveryPhraseDom, 'p')[1];
-    expect(mnemonic.textContent).toBe('');
-  });
+    expect(getRenderedMnemonic(recoveryPhraseDom).textContent).toBe('');
+  }, 60000);
 
-  it('shows the mnemonic for the current Persona', async (): Promise<void> => {
-    const accountDom = await processSignUp(store);
-    await whenOnNavigatedToRoute(store, ACCOUNT_STATUS_ROUTE);
-
-    //from account view, click on hamburger button
-    const hamburgerButton = TestUtils.scryRenderedDOMComponentsWithTag(accountDom, 'button')[0];
-    expect(hamburgerButton.getAttribute('aria-label')).toBe('Open drawer');
-    TestUtils.act(
-      (): void => {
-        TestUtils.Simulate.click(hamburgerButton);
-      },
-    );
-
-    //then click on show recovery phrase
-    const hamburgerList = await findRenderedDOMComponentWithId(accountDom, DRAWER_HTML_ID);
-    const hamburgerElements = (hamburgerList as Element).querySelectorAll('div > div:nth-of-type(2)');
-    if (!hamburgerElements) {
-      throw new Error();
-    }
-    expect(hamburgerElements.length).toBe(2);
-    const recoveryPhraseLink = hamburgerElements[0];
-    expect(recoveryPhraseLink.textContent).toBe('Recovery words');
+  it('shows the mnemonic for the current Persona', async () => {
+    recoveryPhraseDom = await travelToRecoveryPhrase(store, personaMock);
+    expect(getRenderedMnemonic(recoveryPhraseDom).textContent).toBe(mnemonic);
+  }, 60000);
 
-    TestUtils.act(
-      (): void => {
-        TestUtils.Simulate.click(recoveryPhraseLink);
-      },
-    );
+  it('shows the mnemonic for the current Persona when accessed through Drawer menu', async () => {
+    const accountDom = await travelToAccount(store, personaMock);
 
+    mockGetPersonaData(personaMock);
+    await Drawer.clickRecoveryPhrase(accountDom);
```

Comment:
As far as I remember you placed inside `src/routes/accounts/test` the same helper, right? Please use it (I know we are in a separate route) and delete the `utils/test/drawer.ts`